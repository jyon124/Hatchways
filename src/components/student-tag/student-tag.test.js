describe("Student Tag Component Tests", () => {
    it.todo("Renders a Student Tag component");
    it.todo("Test if the tag is removed when its clicked");
});
