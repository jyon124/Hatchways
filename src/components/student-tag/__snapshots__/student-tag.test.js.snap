// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Student Tag Component Tests Renders a Student Tag component 1`] = `"<div class=\\"tag\\">test<span class=\\"tagDelete\\">X</span></div>"`;
